### PR TITLE
fix(serverless): point serverless flags at different modesl since they get installed later

### PR DESCRIPTION
**Pull request by joshuawilson**
To test this you need to either start with a clean install or completely remove the Serverless operator.

Then on the admin console you can go to the Serverless nav section and select anything there. 

Fixes https://jira.coreos.com/browse/ODC-1477

### CODE REVIEW

**Comment by joshuawilson** [comment]
@invincibleJai this simple fix appears to work from my testing. Can you please try it.

**Comment by invincibleJai** [comment]
@joshuawilson it seems to work, am not seeing 404. The issue with relying on `knativeServing` CRDs was like it used to there even before controls were created and so for like 10 sec or so we see 404s and this doesn't seem to be the case with other CRDs i.e Service/Routes/Revisions etc.

```
kubectl get crd | grep knative
certificates.networking.internal.knative.dev                2019-06-26T11:55:21Z
clusteringresses.networking.internal.knative.dev            2019-06-26T11:55:22Z
configurations.serving.knative.dev                          2019-06-26T11:55:22Z
images.caching.internal.knative.dev                         2019-06-26T11:55:22Z
knativeservings.serving.knative.dev                         2019-06-26T07:15:45Z
podautoscalers.autoscaling.internal.knative.dev             2019-06-26T11:55:23Z
revisions.serving.knative.dev                               2019-06-26T11:55:23Z
routes.serving.knative.dev                                  2019-06-26T11:55:24Z
serverlessservices.networking.internal.knative.dev          2019-06-26T11:55:24Z
services.serving.knative.dev                                2019-06-26T11:55:24Z
```

I would suggest introducing 3 more Feature flag i.e `FLAG_KNATIVE_SERVING_SERVICE`, `FLAG_KNATIVE_SERVING_ROUTE`, `FLAG_KNATIVE_SERVING_REVISION` associated with their respective CRDs and have them associated with respective navitems i.e 

```
{
    type: 'FeatureFlag/Model',
    properties: {
      model: models.RevisionModel,
      flag: FLAG_KNATIVE_SERVING_REVISION,
    },
 } 

....

{
    type: 'NavItem/ResourceNS',
    properties: {
      section: 'Serverless',
      componentProps: {
        name: models.RevisionModel.labelPlural,
        resource: referenceForModel(models.RevisionModel),
        required: FLAG_KNATIVE_SERVING_REVISION,
      },
    },
 }
```

Similarly for Routes/Services as it's good if they rely on respective CRDs. WDYT?

**Comment by joshuawilson** [comment]
@invincibleJai I have added the other flags. It is more complete now.

File: frontend/packages/knative-plugin/src/plugin.tsx [hunk 1]
```diff
@@ -31,13 +37,41 @@ const plugin: Plugin<ConsumedExtensions> = [
       models: _.values(models),
     },
   },
+  {
+    type: 'FeatureFlag/Model',
+    properties: {
+      model: models.ConfigurationModel,
+      flag: FLAG_KNATIVE_SERVING_CONFIGURATION,
```
Comment (invincibleJai) [inline frontend/packages/knative-plugin/src/plugin.tsx]: `FLAG_KNATIVE_SERVING_CONFIGURATION` we are not using as of now still no harm in having it.

Comment (joshuawilson) [reply frontend/packages/knative-plugin/src/plugin.tsx]: I thought it was worth it since we were adding the others. Just for completeness.

**Pull request merged**